README.md

```diff
@@ -23,52 +32,77 @@ import (
 )
 
 type Config struct {
-	ListenPort int    `json:"listen_port"`
-	Debug      bool   `json:"debug"`
-	LogFile    string `json:"log_file"`
+	
+	Field1       string `json:"field1"`
+	Field2       int    `json:"field2"`
+
+	// This field can be obfuscated with the structviewer tag
+	Field3secret string `json:"field3_secret" structviewer:"obfuscate"` 
 }
 
 func main() {
-	config := &struct_viewer.Config{
-		Object: &Config{
-			ListenPort: 8080,
-			Debug:      true,
-			LogFile:    "/var/log/app.log",
-		},
-		Path:          "./main.go",
-		ParseComments: true,
-	}
+
+	  // Define configuration structure
+  	config := &structviewer.Config{
+               Object: &YourConfigStruct{
+   		       Field1: "value1",
+  		       Field2: 123,
+   	       },
+
+        }
 
 	// prefix is added to each env var
-	v, err := structviewer.New(config, "APP_")
+	v, err := structviewer.New(config, "PREFIX")
 	if err != nil {
 		panic(err)
 	}
 
 	http.HandleFunc("/config", v.JSONHandler)
 	http.HandleFunc("/envs", v.EnvsHandler)
-	log.Println("Running server on port :8080")
 	err = http.ListenAndServe(":8080", nil)
 	if err != nil {
 		log.Fatal("ListenAndServe: ", err)
 	}
+	log.Println("Running server on port :8080")
```

Comment:
WIll this be printed? If I remember correctly, this line:
```
err = http.ListenAndServe(":8080", nil)
```
blocks the execution of the program at that step if I'm not wrong.